chore: Update integrations spec reflect `EigenDACertVeriferRouter` conception and tightened glossary term usage (#1642)
Layr-Labs/eigenda: docs/spec/src/integration/spec/6-secure-integration.md

## Validity Conditions

EigenDA is a service that assures the availability and integrity of payloads posted to it for 14 days.
When deriving a rollup chain by running its derivation pipeline, only EigenDA `DACerts` that satisfy the three validity conditions are considered valid and used:
1. RBN Recency Validation - ensure that the DA Cert's reference block number (RBN) is not too old with respect to the L1 block at which the cert was included in the rollup's batcher-inbox. This ensures that the blob on EigenDA has sufficient availability time left (out of the 14 day period) in order to be downloadable if needed during a rollup fault proof window.
2. Cert Validation - ensures sufficient operator stake has signed to make the blob available, for all specified quorums. The stake is obtained onchain at a given reference block number (RBN) specified inside the cert.
3. Blob Validation - ensures that the blob used is consistent with the KZG commitment inside the Cert.

> If a certificate fails validation then it is disregarded from the rollup's derivation pipeline and treated as an empty entry.

Reviewer: ```suggestion
If a certificate fails validation then it MUST be discarded from the rollup's derivation pipeline.
```

Reviewer: actually blob validation failing shouldnt lead to discarding of cert right...? cc @bxue-l2 
Because that just means the relay or validators are being malicious, but the cert itself is correct. it just points to another blob which we need to wait for (this should stall the derivation pipeline I think?)

Author: feel free to own @bxue-l2

### 2. Cert Validation

Cert validation is done inside the EigenDACertVerifier contract, which EigenDA deploys as-is, but is also available for rollups to modify and deploy on their own. Specifically, [checkDACert](https://github.com/Layr-Labs/eigenda/blob/2414ed6f11bd28bc631eab4da3d6b576645801b0/contracts/src/periphery/cert/EigenDACertVerifier.sol#L46-L56) is the entry point for validation. This could either be called during a normal eth transaction (either for pessimistic “bridging” like EigenDA V1 used to do, or when uploading a Blob Field Element to a one-step-proof’s [preimage contract](https://specs.optimism.io/fault-proof/index.html#pre-image-oracle)), or be zk proven using a library like [Steel](https://github.com/risc0/risc0-ethereum/blob/main/crates/steel/docs/what-is-steel.md).

> `checkDACert` takes in an unstructured `[]byte` input. This is done to ensure seamless upgrades where the underlying representation of the `DACert` can change over time with new version introductions.

Reviewer: prob need to mention that this is actually only to support the router interface not changing, otherwise this is confusing because CertVerifier in itself is immutable and can't change over time

Reviewer: also please mention abi-encoded somewhere, not just "unstructured []byte"

|                     | Nitro V1       | OP V1 (insecure) | Nitro V2       | OP V2                                                                                |
| ------------------- | -------------- | ---------------- | -------------- | ------------------------------------------------------------------------------------ |

> | Cert Verification   | SequencerInbox | x                | one-step proof | one-step proof: done in one step proof HostIO contract |

Reviewer: you changed OP column instead of arb column I think? OP doesn't have a HostIO contract
